Library/ViewModels/LandingViewModel.swift

```diff
@@ -15,7 +16,12 @@ public protocol LandingViewModelType {
 }
 
 public final class LandingViewModel: LandingViewModelType, LandingViewModelInputs, LandingViewModelOutputs {
+  private let hasSeenCategoryPersonalizationFlowProperty = MutableProperty(false)
```

Comment:
This property is just here to make sure that our side-effect closure doesn't get deallocated. Since it's not used outside the scope of this view model, it's marked as `private`.

Reply:
I think here you can simply do: 
```
self.viewDidLoadProperty.signal
      .observeValues {
        AppEnvironment.current.userDefaults.hasSeenCategoryPersonalizationFlow = true
      }
```